src/core/variable.jl

```diff
@@ -671,10 +943,75 @@ function variable_mc_generation_reactive(pm::_PMs.AbstractPowerModel; nw::Int=pm
         end
     end
 
+    _PMs.var(pm, nw)[:qg_bus] = Dict{Int, Any}()
+
     report && _PMs.sol_component_value(pm, nw, :gen, :qg, _PMs.ids(pm, nw, :gen), qg)
```

Comment:
Do you not want to automatically report pg_bus and qg_bus?

Reply:
`pg_bus` and `qg_bus` only get defined in the constraint calls, so I cannot add them here; instead, I add them once they are created in the constraint calls.